action/Dockerfile.dev

```diff
@@ -1,4 +1,5 @@
-FROM docker.io/node:12-alpine
+#FROM docker.io/node:12-alpine
```

Comment:
Should we pin this to a SHA? I remember we got bit in RulesDecision as the version got updated when we built and deployed.